Build.PL

```diff
@@ -44,13 +46,13 @@ my $builder = Alien::Base::ModuleBuild->new(
     ## For development, to save the download time,
     ## download the package once and put it in an 'inc' directory.
     # alien_repository => {
-    #                      protocol => 'local',
-    #                      location => 'inc',
-    #                      exact_filename => 'ImageMagick.tar.gz',
-    #                     },
+    #                     protocol => 'local',
+    #                     location => 'inc',
+    #                     exact_filename => 'ImageMagick.tar.gz',
+    #                    },
     alien_build_commands => [
                              ## This siteprefix will install the execs in perl's /bin and the libs in perl's lib/
-                             'LDFLAGS='.$LDFLAGS.' %pconfigure --prefix=%s --exec-prefix='.$siteprefix.' --with-perl='.$perlbin.' '.$enable_shared.' '.$use_threads ,
+                             'LDFLAGS='.$LDFLAGS.' %c --prefix=%s --exec-prefix='.$siteprefix.' --with-perl='.$perlbin.' '.$enable_shared.' '.$use_threads ,
```

Comment:
what's the diff between %pconfigure and %c ?

Reply:
it is the same thing. %pconfigure is currently deprecated, in favor of %c.